app/PharBuilder.php

```diff
@@ -133,6 +133,9 @@ protected function buildPhar()
         $dirsFiles = $this->readComposerAutoload();
         $this->output->writeln(' <info>OK</info>');
 
+        // Unlink, otherwise we just add things to the already existing phar
+        unlink($this->pharName);
```

Comment:
You must test if the file exist or not before trying to remove it.
(It throw a `Warning: unlink(./phar-builder.phar): No such file or directory in /home/you/PharBuilder/app/PharBuilder.php on line 137`)